### PR TITLE
Add `env_logger` to tour example

### PR DESCRIPTION
**Pull request by TGotwig**
Relates #7

### CODE REVIEW

File: src/logger.rs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+// use flexi_logger::{Logger, opt_format};
+impl Logger {
+    /// Creates a new [`Logger`].
+    ///
+    /// [`Logger`]: struct.Logger.html
+    pub fn log(msg: &str) {
+        let now: DateTime<Utc> = Utc::now();
+        info!("{}: {}\n", now, msg);
+    }
+}
```
Comment (hecrj) [inline src/logger.rs]: This is normally implemented by a particular logging implementation. For instance, [`env_logger`](https://docs.rs/env_logger/0.6.2/env_logger/) already prints the time.

I think we do not need a `Logger`. We can use `log` macros directly, once we need them.

File: examples/tour/main.rs [hunk 2]
```diff
@@ -33,6 +37,19 @@ pub fn main() -> ggez::GameResult {
...
+            TermLogger::new(LevelFilter::Warn, Config::default(), TerminalMode::Mixed).unwrap(),
+            WriteLogger::new(LevelFilter::Info, Config::default(),
+                OpenOptions::new()
+                    .append(true)
+                    .create(true)
+                    .open(".log")
+                    .unwrap()
+            ),
+        ]
+    ).unwrap();
```
Comment (hecrj) [inline examples/tour/main.rs]: We should probably initialize the logger before building the context.

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,6 @@
+.*
+!/.gitignore
+!/.travis.yml
```
Comment (hecrj) [inline .gitignore]: Ignoring all hidden files by default seems like a bit too much 😅 

You can ignore files locally by editing: `.git/info/exclude`.

File: Cargo.toml [hunk 4]
```diff
@@ -20,6 +20,9 @@ features = ["winit"]
 [dependencies]
 stretch = "0.2"
 twox-hash = "1.5"
+chrono = "0.4.9"
+log = "0.4.8"
+simplelog = "0.7.2"
```
Comment (hecrj) [inline Cargo.toml]: I think [`env_logger`](https://docs.rs/env_logger/0.6.2/env_logger/) is a simpler alternative for the example. We should also add it as a development dependency instead.

File: src/logger.rs (hunk 1 above)
Comment (TGotwig) [reply src/logger.rs]: Ah yeah right, forgot to edit this 🙈

File: .gitignore (hunk 3 above)
Comment (TGotwig) [reply .gitignore]: All hidden files except `.gitignore` and `.travis.yml` 😉
But yeah maybe it's better to use `.git/info/exclude` for that 🙂

File: Cargo.toml (hunk 4 above)
Comment (TGotwig) [reply Cargo.toml]: Hmm but `env_logger` can't write a log-file 😬

Comment (hecrj) [reply Cargo.toml]: You are right! However, I think it is not a necessary feature for the example and we should probably try to keep it as simple and straightforward as possible.

**Comment by TGotwig** [comment]
Did all the changes, squashed, and force-pushed 😇

File: Cargo.toml [hunk 5]
```diff
@@ -20,6 +20,7 @@ features = ["winit"]
 [dependencies]
 stretch = "0.2"
 twox-hash = "1.5"
+log = "0.4.8"
```
Comment (hecrj) [inline Cargo.toml]: We are not using the dependency for anything yet... We should probably add it once we need it.

File: examples/tour/main.rs [hunk 6]
```diff
@@ -12,6 +12,8 @@ use ggez::filesystem;
 use ggez::graphics;
 use ggez::input::mouse;
 
+extern crate env_logger;
```
Comment (hecrj) [inline examples/tour/main.rs]: `extern crate` is not necessary when using the 2018 edition of Rust.

File: examples/tour/main.rs [hunk 7]
```diff
@@ -33,6 +35,7 @@ pub fn main() -> ggez::GameResult {
         true,
     );
 
+    env_logger::init();
```
Comment (hecrj) [inline examples/tour/main.rs]: This should probably be the first thing we do in `main` :)

**Pull request merged**